internal/pkg/cli/pipeline_init.go

```diff
@@ -296,9 +295,20 @@ func (o *initPipelineOpts) createBuildspec() (string, error) {
 		return "", err
 	}
 	path, err := o.workspace.WritePipelineBuildspec(content)
+	var buildspecExists bool
 	if err != nil {
-		return "", fmt.Errorf("write buildspec to workspace: %w", err)
+		e, ok := err.(*workspace.ErrFileExists)
+		if !ok {
+			return "", fmt.Errorf("write buildspec to workspace: %w", err)
+		}
+		buildspecExists = true
+		path = e.FileName
+	}
+	buildspecMsgFmt := "Wrote the buildspec for the pipeline's build stage at '%s'\n"
+	if buildspecExists {
+		buildspecMsgFmt = "Buildspec file for project pipeline already exists at %s, skipping writing it.\n"
```

Comment:
What do you think of `color.Emphasize`-ing "pipeline already exists"? 
https://github.com/aws/amazon-ecs-cli-v2/blob/1e58c0ddbbd61ce1d2890f24f936d0fcf8efbeed/internal/pkg/term/color/color.go#L54-L57

Reply:
hmm, we don't do that for the app manifest already exists message, and it seems like we mostly emphasize things that require user input. I could go either way, but I feel like it might be overkill?